app/src/main/java/de/westnordost/streetcomplete/quests/surface/AddCyclewayPartSurface.kt

```diff
@@ -8,7 +8,13 @@ import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataDao
 class AddCyclewayPartSurface(o: OverpassMapDataDao) : SimpleOverpassQuestType<String>(o) {
 
     override val tagFilters = """
-        ways with highway ~ path|footway|cycleway|bridleway and surface = paved and bicycle ~ designated|yes and segregated=yes
+        ways with
+        (
+          highway = cycleway 
+          or (highway ~ path|footway and bicycle != no)
```

Comment:
I assume this will also match `highway=path` with no bicycle key whatsoever?

Reply:
Yes